### PR TITLE
ArticleList

### PR DESCRIPTION
**Pull request by omar-baggio**

### CODE REVIEW

File: Utils/api.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const ncNews = axios.create({
+  baseURL: "https://nc-news-api-w4qo.onrender.com/api",
+});
+
+export const getArticles = () => {
+  let path = "/articles";
+
+  return ncNews.get(path).then(({ data: { articles } }) => {
+    console.log(articles, "<--- res in api");
```
Comment (B7Bprog) [inline Utils/api.js]: Please remove all logs.

File: src/components/ArticlesList.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../Utils/api";
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((dataFromApi) => {
+      console.log(dataFromApi, "<--- articles in articleList");
```
Comment (B7Bprog) [inline src/components/ArticlesList.jsx]: Don't need this log either.

File: src/components/ArticlesList.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../Utils/api";
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
```
Comment (B7Bprog) [inline src/components/ArticlesList.jsx:6]: Please add a state for errors and one for a 'loading' indication and please make sure it displays those messages in the appropriate cases.

File: src/components/ArticlesList.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      setArticles(dataFromApi);
+    });
+  }, []);
+
+  return (
+    <>
+      <h1>Article List</h1>;
+      {articles.map((article) => {
+        return (
+          <section key={article.article_id} className="articlesbox">
```
Comment (B7Bprog) [inline src/components/ArticlesList.jsx]: This part could be extracted into a separate component.

File: src/App.css [hunk 5]
```diff
@@ -5,4 +5,26 @@
...
+
+  border: 2px solid red;
+
+}
 
+.imgList {
+  max-height: 200px;
+  padding-top: 30px;
+  margin: 10px;
+}
```
Comment (B7Bprog) [inline src/App.css]: Please think about a responsive design, where the screen size is considered.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Omar, please see my comments, refactor your code, then send us another PR with it.

**Comment by NRMSMITH** [comment]
Excellent implementation of the feedback ! I'm happy for you to merge this 👍

**Pull request merged**